### PR TITLE
Updated DynamoDB Adaptor documentation

### PR DESCRIPTION
**Pull request by manishprivet**
## Reasoning 💡
#90 changed the way DynamoDB was implemented, and the documentation needed to be updated accordingly.

## Checklist 🧢

- [x] Documentation
- [ ] Tests
- [ ] Ready to be merged

## Affected issues 🎟
Addresses #125

### CODE REVIEW

**Review by balazsorban44 — CHANGES_REQUESTED** [review]
So in the source I see that table name has a default value, so it is actually not needed for a default setup. It defaults to `"next-auth" `

File: packages/dynamodb/README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ npm install next-auth @next-auth/dynamodb-adapter@canary
 
 2. Add this adapter to your `pages/api/[...nextauth].js` next-auth configuration object.
 
-You need to pass `aws-sdk` to the adapter in addition to the table name.
+You need to pass `DocumentClient` from `aws-sdk` to the adapter in addition to the table name.
```
Comment (balazsorban44) [inline packages/dynamodb/README.md:33]: Same as https://github.com/nextauthjs/next-auth/pull/2125#discussion_r645206437

**Review by balazsorban44 — CHANGES_REQUESTED** [review]

**Review by balazsorban44 — CHANGES_REQUESTED** [review]
I would rather not mention table name this way, just show the bare minimal implementation, and maybe add an options section in the docs, under schema for example.

**Comment by ndom91** [comment]
Looks good to me!

**Comment by stephankaag** [comment]
Hi @manishprivet 

With the current table design sessions cannot be looked up with the session's AccessToken. Is that on purpose? If so: what's the point of the AccessToken?

**Comment by manishprivet** [comment]
> Hi @manishprivet 
> 
> With the current table design sessions cannot be looked up with the session's AccessToken. Is that on purpose? If so: what's the point of the AccessToken?

I didn't work on the table design actually. I think @PolMrt could answer your question

**Pull request merged**